examples/universal/z_get.cxx

```diff
@@ -42,7 +42,7 @@ int _main(int argc, char **argv) {
     }
 #endif
 
-    ZError err;
+    ZResult err;
```

Comment:
ZResult err, should likely also be changed to ZResult res

Reply:
This is a too broad and risky change throughout the codebase since it will have impact on the correctness of the code. For being on the safe side we have decided the variable renaming at this stage.